extension/src/setup/index.ts

```diff
@@ -374,7 +374,8 @@ export class Setup
     return this.sendDataToWebview()
   }
 
-  public getDvcCliDetails(): DvcCliDetails {
+  private async getDvcCliDetails(): Promise<DvcCliDetails> {
+    await this.config.isReady()
```

Comment:
[F] Need to make sure that the config has set the pythonBinPath before we try to access it.